benchmark/run_benchmark.py

```diff
@@ -416,15 +417,19 @@ def run_benchmark(config):
         # ==================================================================
         # GCC vs GCC string for month and year (e.g. "Jul2016")
         # ==================================================================
-        if np.equal(gcc_ref_date, gcc_dev_date):
-            mon_yr_str = calendar.month_abbr[gcc_ref_b_start[1]] + gcc_ref_s_start[0]
+        if np.equal(gcc_ref_date, gcc_dev_date) and np.equal(
```

Comment:
Is the header above this section ("Jul2016") still accurate? It looks like the new string will be a date range rather than month/year. Same comment for the other sections that have this.

Reply:
Good catch -- fixed!